toxcore/tox_events.c

```diff
@@ -102,8 +102,8 @@ Tox_Events *tox_events_iterate(Tox *tox, bool fail_hard, Tox_Err_Events_Iterate
     return state.events;
 }
 
-non_null(1) nullable(2, 3)
-static bool tox_events_pack(Bin_Pack *bp, const Logger *logger, const void *obj)
+non_null(2) nullable(1, 3)
+static bool tox_events_pack(const void *obj, const Logger *logger, Bin_Pack *bp)
```

Comment:
Those `non_null()` and `nullable()` don't match the behavior before the change. Should be `non_null(3) nullable (1, 2)` to match the previous behavior. Was that intentional?

Reply:
Yes, luckily cimple warns about it. Fixed.

Reply:
I don't see it being fixed, looks like nothing has changed. Did you forget to stage the hunk or something?

Reply:
Fixed now.